docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md

```diff
@@ -0,0 +1,205 @@
+
+- `first_touch`: Assigns 100% attribution to the first channel in each path.
+- `last_touch`: Assigns 100% attribution to the last channel in each path.
+- `position_based`: The first and last channels get 40% of the credit each, with the remaining channels getting the leftover 20% distributed evenly.
+- `linear`: Assigns attribution evenly between all channels on the path.
+​
+<p align="center">
+<ThemedImage
+alt='Data processing model for the attribution package'
+sources={{
+  light: require('./images/attribution_models_light.png').default,
+  dark: require('./images/attribution_models_dark.png').default
+}}
+/>
+</p>
+
+
+| COMPOSITE_KEY                                       | EVENT_ID                             | CUSTOMER_ID          | CV_TSTAMP               | CV_TOTAL_REVENUE | CHANNEL_TRANSFORMED_PATH         | CHANNEL        | SOURCE_INDEX | PATH_LENGTH | FIRST_TOUCH_ATTRIBUTION | LAST_TOUCH_ATTRIBUTION | LINEAR_ATTRIBUTION | POSITION_BASED_ATTRIBUTION |
+|-----------------------------------------------------|--------------------------------------|----------------------|-------------------------|------------------|----------------------------------|----------------|--------------|-------------|-------------------------|------------------------|--------------------|----------------------------|
+| 9f80548b-5bb3-4d21-b78e-503ae6f8205eDirect0         | 9f80548b-5bb3-4d21-b78e-503ae6f8205e | f000170187170673177  | 2023-06-08 20:18:32.000 | 50               | Direct                           | Direct         | 0            | 1           | 50                      | 50                     | 50                 | 50                         |
+| f9e6fa53-61f4-45f4-adb1-b75e018f8a32Direct0         | f9e6fa53-61f4-45f4-adb1-b75e018f8a32 | f0009028775170427694 | 2023-06-11 15:33:03.000 | 20.42            | Direct                           | Direct         | 0            | 1           | 20.42                   | 20.42                  | 20.42              | 20.42                      |
+| c3bf5de5-20c7-42d6-9543-afd9ecf133f5Video0          | c3bf5de5-20c7-42d6-9543-afd9ecf133f5 | f0008284662789123943 | 2023-07-07 13:05:55.000 | 200              | Video                            | Video          | 0            | 1           | 200                     | 200                    | 200                | 200                        |
+| 1487c09f-f17f-4c76-990e-55c46f8f621cDisplay_Other0  | 1487c09f-f17f-4c76-990e-55c46f8f621c | f0006911334202687206 | 2023-07-19 04:27:51.000 | 66.5             | Display_Other > Organic_Search   | Display_Other  | 0            | 2           | 66.5                    | 0                      | 33.
```

Comment:
yeah, I deleted some extra lines otherwise it would have been too long so it won't make sense in the total context but will have a look to add that just so the logic is easier to understand.

Reply:
added the extra line you were referring to, agreed, let's make it clear to understand!